cdap-api/src/main/java/co/cask/cdap/internal/security/SecureStoreProvider.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.internal.security;
+
+/**
+ * This is tagging interface for a class providing an implementation of the secure store.
+ * TODO: Add methods to this once another implementation is added.
+ */
+public interface SecureStoreProvider {
```

Comment:
Why it is just a tagging interface? Shouldn't it at least have a `get` method if it is a Provider?